bowling/src/lane.cpp

```diff
@@ -1,5 +1,34 @@
 #include "lane.hpp"
 
-Lane::Lane() {}
+#include <filesystem>
+
+// private
+
+void Lane::parseFile(std::string fileName)
+{
+    std::string fileExtension = std::filesystem::path(fileName).extension();
```

Comment:
please consider using such contruction in the begging of file:
`namespace fs = std::filesystem;`
You could use:
`fs::path()`
with this construction

Reply:
i dont know if its necessary because i use it just one time

Reply:
ok, you can keep it this way :)